app/components/ChallengeDetailComponents/CoordinatesView.js

```diff
@@ -52,15 +53,15 @@ export default class CoordinatesView extends Component {
 					 </TouchableHighlight>
 					:
 					<TouchableHighlight  key={location.title+location.longitude+location.latitude}>
-					 	<View style = {{flex:1, flexDirection: 'row',}} >
+					 	<View style = {{flex:1, flexDirection: 'row', height: 55,alignItems:'center',}} >
```

Comment:
Now that your styles have grown I think you should make a stylesheet that has all your styles. This will separate your styling from your view, making your view easier to read, and the styles easier to find/modify.